src/three/TilesRenderer.js

```diff
@@ -615,8 +615,12 @@ export class TilesRenderer extends TilesRendererBase {
 			// any transformations applied to it can be assumed to be applied after load
 			// (such as applying RTC_CENTER) meaning they should happen _after_ the z-up
 			// rotation fix which is why "multiply" happens here.
-			scene.matrix.multiply( tempMat ).premultiply( cachedTransform );
-			scene.matrix.decompose( scene.position, scene.quaternion, scene.scale );
+			if ( extension !== 'pnts' ) {
+
+				scene.matrix.multiply( tempMat ).premultiply( cachedTransform );
+				scene.matrix.decompose( scene.position, scene.quaternion, scene.scale );
+
+			}
```

Comment:
The only matrix we don't want to apply is the `tempMat` matrix when a model is made from points -- the `cachedTransform` matrix still needs to be applied because it represents and offset transformation from the tileset tiles. Perhaps the cachedTransform multiplication and decompose can be applied after this condition?

```js

// ...

if ( extension !== 'pnts' ) {

	scene.matrix.multiply( tempMat );

}

scene.matrix.premultiply( cachedTransform );
scene.matrix.decompose( scene.position, scene.quaternion, scene.scale );

// ...
```

Reply:
Done